### ISSUE TITLE
Ch 12: merge

### ISSUE THREAD

**Issue by dfolch** [gdsbook/book#319]
Chapter 12 typo? The default for merge is intersection, but I think the goal here is to keep all the observations. If this is the correct interpretation, then the resulting quantile map would likely need to be updated too since there are so many zeros.

Original
---
```python
airbnbs_w_counts = airbnbs_albers.merge(poi_count, left_on="id", right_index=True).fillna({"poi_count": 0})
```
This results in 4601 observations. 

Revision
---
```python
airbnbs_w_counts = airbnbs_albers.merge(poi_count, left_on="id", how='left', right_index=True).fillna({"poi_count": 0})
```
This results in 6110 observations.

**Comment by ljwolf** [comment]
Yes, thanks for noticing this, should have probably been a left join. We've updated this now in #318
